### ISSUE TITLE
Getting error when using tab to see what can be listed

### ISSUE THREAD

**Issue by lucasmz-dev** [igo95862/bubblejail#151]
### Output of `bubblejail --version`

0.9.4

### Your distro name and version

Fedora 41 - installed from secureblue/bubblejail COPR

### Description

When clicking Tab to show what can be listed in 0.9.4, an error shows up.

```bash
Traceback (most recent call last):
  File "/usr/bin/bubblejail", line 20, in <module>
    bubblejail_main()
    ~~~~~~~~~~~~~~~^^
  File "/usr/lib64/bubblejail/python-packages/bubblejail/bubblejail_cli.py", line 218, in bubblejail_main
    run_autocomplete()
    ~~~~~~~~~~~~~~~~^^
  File "/usr/lib64/bubblejail/python-packages/bubblejail/bubblejail_cli_autocomplete.py", line 116, in run_autocomplete
    for x in AutoCompleteParser().auto_complete(args.current_cmd):
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/bubblejail/python-packages/bubblejail/bubblejail_cli_autocomplete.py", line 103, in auto_complete
    self.auto_complete_parser(current_cmd)
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/usr/lib64/bubblejail/python-packages/bubblejail/bubblejail_cli_autocomplete.py", line 83, in auto_complete_parser
    self.last_auto_complete = iter_list_choices()
                              ~~~~~~~~~~~~~~~~~^^
  File "/usr/lib64/bubblejail/python-packages/bubblejail/bubblejail_cli.py", line 40, in iter_list_choices
    assert isinstance(choices, set)
           ~~~~~~~~~~^^^^^^^^^^^^^^
AssertionError
```

Steps to reproduce:
1. Go into a terminal
2. Write `bubblejail list`
3. Click tab to show what can be listed

**Comment by igo95862** [comment]
This is probably caused by 28f7816bbde4c5f4091ac4122112fc9c9220770e.

I will try to make a hotfix as soon as possible.

**Comment by igo95862** [comment]
It is also affected by the `-O` optimization level as it removes assertions.

**Comment by igo95862** [comment]
I pushed b48051440315b69d5edef43259e4c8a1bfcca983. Can you give it a test?

**Comment by igo95862** [comment]
What kind of issues?

**Comment by igo95862** [comment]
Fixed in 0.9.4.1

**Comment by lucasmz-dev** [comment]
> What kind of issues?

Sorry about that just busy with christmas time, I don't remember exactly, sorry about not trying to repro, I don't have access to my PC atm
